xactserver/proto/rpc.proto

```diff
@@ -11,15 +11,22 @@ message PrepareRequest {
 message PrepareResponse {
 }
 
-enum Vote {
-    COMMIT = 0;
-    ABORT = 1;
-}
-
 message VoteRequest {
```

Comment:
Shouldn't this be VoteResponse?

Reply:
The Request and Response suffixes are just to follow gRPC's convention when defining messages for the procedures (e.g. `rpc Prepare(PrepareRequest) returns (PrepareResponse);`).

If we use gRPC the "right" way, the `PrepareResponse` should be `VoteResponse` and, we need to return the vote result inside the implementation of the Prepare rpc. However, I found it more natural to implement the xactserver with a message passing model than using gRPC the way it is (e.g. after preparing, the current node has to broadcast the vote to all other participants instead of just this single response). Therefore, we only utilize the Requests to pass messages to other nodes and leave the Responses empty.

Reply:
Does grpc enforce the naming? If it is not enforced, how about you just rename it as "Vote"? The VoteRequest part makes it confusing.

Reply:
Yeah, I agree. I'll rename them in a separate PR.